addons/website_slides/views/website_courses_templates.xml

```diff
@@ -0,0 +1,623 @@
+<?xml version="1.0" ?>
+<odoo>
+<!-- <template id="courses" name="Courses">
```

Comment:
Commented code to remove if not necessary anymore.

Reply:
Done